src/Tensors.jl

```diff
@@ -156,8 +154,8 @@ for TensorType in (SymmetricTensor, Tensor)
     end
 end
 
-# Binary op(Tensor, Tensor): +, -, .+, .-, .*, ./
-for op in (:+, :-, :.+, :.-, :.*, :./)
+# Binary op(Tensor, Tensor): +, -
+for op in (:+, :-)
```

Comment:
That was not supported anyway.
No matter what, these should be implemented with `broadcast`. Can be a separate PR.